contribs/gmf/examples/locationchooser.js

```diff
@@ -43,10 +43,10 @@ app.MainController = function() {
 
 
   /**
-   * @type {gmfx.LocationchooserLocation}
+   * @type {string}
    * @export
    */
-  this.selectedLocation = this.locations[2];
+  this.selectedLocation = '[' + this.locations[2].extent + ']';
```

Comment:
Ok I think I understand. This is related to your comment in the issue description and the first "yellow" note in the Angular documentation for "select": https://docs.angularjs.org/api/ng/directive/select.